src/components/BlogForm/types.ts

```diff
@@ -1,7 +1,7 @@
-type DataProps = {
+export type DataProps = {
   title: string;
   desc: string;
   tags: string;
 };
 
-export type { DataProps };
+export type titleDataProps = { title: string };
```

Comment:
If the type is just one key I would use that instead of creating and exporting it. Also for naming convention props are used for components not in functions.

```const myFunc = (data: {title: string}) => {}```